lib/src/scope.dart

```diff
@@ -46,7 +46,7 @@ class Scope {
   /// Calls an isolate-scoped RPC named [method] with [params].
   ///
   /// This always adds the `isolateId` parameter to the RPC.
-  Future<Object> sendRequest(String method, [Map<String, Object> params]) async {
+  Future<dynamic> sendRequest(String method, [Map<String, Object> params]) async {
```

Comment:
Hmm, I see. In practice, I think all VM service responses are Maps, so you should be able to at least type is as `Map<String, dynamic>` and include an `as` check in the body.